### PR TITLE
#23 - Implement adding archive in asto repository

### PR DESCRIPTION
**Pull request by genryxy**
Part of #23 
Added implementation for uploading archive in `AstoRepository` and tests

### CODE REVIEW

File: src/main/java/com/artipie/composer/http/AddArchiveSlice.java [hunk 1]
```diff
@@ -75,7 +76,18 @@ public Response response(
         final Matcher matcher = AddArchiveSlice.PATH.matcher(uri);
         final Response resp;
         if (matcher.matches()) {
-            throw new NotImplementedException("not implemented yet");
+            final Content content = new Content.From(body);
+            resp = new AsyncResponse(
+                this.repository
+                    .addArchive(
+                        new Archive.Zip(
+                            new Archive.Name(matcher.group(1), matcher.group("version")),
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/composer/http/AddArchiveSlice.java:84]: @genryxy let's address to the regex groups in the same style here - either by group name or by number

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy thanks, just one suggestion

**Comment by genryxy** [comment]
@olenagerasimova thanks, used group name for both cases

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks

**Comment by 0crat** [comment]
Job `gh:artipie/composer-adapter#90` is not assigned, can't get performer

**Pull request merged**